cranelift/codegen/src/fx.rs

```diff
@@ -14,7 +14,6 @@ use super::{HashMap, HashSet};
 use core::default::Default;
 use core::hash::{BuildHasherDefault, Hash, Hasher};
 use core::ops::BitXor;
-
```

Comment:
Would you mind reverting this change? I like the space between the imports and type decls :)

Reply:
Oops, leftover from some fiddling to get `Entry` before I found another re-export; sorry about that!